source/wp-content/themes/wporg-developer-2023/src/command-github/style.scss

```
@@ -0,0 +1,56 @@
+.wp-block-wporg-command-github {
+	.github-tracker {
+		margin-bottom: 40px;
+
+		img {
+			width: 23px;
+			position: relative;
+			top: 7px;
+			margin-right: 10px;
+		}
+
+		.btn-group {
+			display: inline;
+		}
+
+		a {
+			text-decoration: none;
+
+			&.button {
+				display: inline-block;
+				margin: 4px 0;
+				border: 1px solid;
+				border-color: #ccc;
+				border-radius: 3px;
+				background: #f7f7f7;
+				color: rgba(0, 0, 0, 0.8);
+				appearance: none;
+				padding: 8px 12px;
+				font-size: var(--wp--preset--font-size--extra-small);
+				box-shadow: 0 1px 0 #ccc;
+				line-height: 1;
+
+				&:hover {
+					border-color: #ccc #bbb #aaa #bbb;
+					background: #eee;
+				}
+
+				.green {
+					color: #46b450;
+				}
+
+				.red {
+					color: #dc3232;
+				}
```

Could these be updated to use the syntax colors? Technically they still fail color contrast on `#f7f7f7`, but it's better than these colors.

```suggestion
				.green {
					color: var(--wp--preset--color--syntax-green);
				}

				.red {
					color: var(--wp--preset--color--syntax-red);
				}
```

In general, I'm surprised this old button style is still being used, I would have expected some kind of update to the new button style (no change requested for button styles, since this does match the mockup).

---

Updated.
I was surprised by that too.
It's a bit inconsistent across the sites, but it doesn't look outdated or out of tune overall to me though.